### PR TITLE
🐛 fix container scanning

### PR DESCRIPTION
**Pull request by imilchev**

### CODE REVIEW

File: explorer/scan/discovery_test.go [hunk 1]
```diff
@@ -253,4 +253,19 @@ func TestDiscoverAssets(t *testing.T) {
 		require.NoError(t, err)
 		assert.Len(t, discoveredAssets.Assets, 1)
 	})
+
+	t.Run("container platform discovery", func(t *testing.T) {
```
Comment (imilchev) [inline explorer/scan/discovery_test.go]: not sure if that's the best place to put this test but it's definitely important to have such tests... let's keep it here for now and we can break off real integration tests to another place later

Comment (imilchev) [reply explorer/scan/discovery_test.go]: oh.. this needs to use a builtin os provider in order to work...

Comment (imilchev) [reply explorer/scan/discovery_test.go]: moved it to the os provider

**Review by chris-rock — APPROVED** [review]
Thank you @imilchev

**Pull request merged**